packages/apollo-server-core/src/runHttpQuery.ts

```diff
@@ -320,7 +364,9 @@ export async function runHttpQuery(
         fieldResolver: optionsObject.fieldResolver,
         debug: optionsObject.debug,
         tracing: optionsObject.tracing,
-        cacheControl: optionsObject.cacheControl,
+        cacheControl: cacheControl
+          ? { defaultMaxAge: cacheControl.defaultMaxAge }
```

Comment:
Is the implication that this is the only parameter that the underlying package supports? maybe use _.omit or whatever to blacklist instead of whitelist?